src/Algolia.Search.Test/Serializer/SerializerTest.cs

```diff
@@ -240,5 +241,13 @@ public void TestPersonalization()
                 "{\"eventsScoring\":{\"Add to cart\":{\"type\":\"conversion\",\"score\":50},\"Purchase\":{\"type\":\"conversion\",\"score\":100}},\"facetsScoring\":{\"brand\":{\"score\":100},\"categories\":{\"score\":10}}}";
             Assert.True(json.Equals(expectedJson));
         }
+
+        [Test]
+        [Parallelizable]
+        public void TestListIndicesResponses64BitsIntegers()
+        {
+            var json = JObject.Parse("{\"entries\": 100000000000, \"dataSize\": 100000000000, \"fileSize\": 100000000000}");
```

Comment:
Nit: Doing `JObject.Parse` and then `json.ToObject` is a bit cumbersome. I'd rather go for a one liner:

```csharp
Assert.DoesNotThrow(() => { JsonConvert.DeserializeObject<IndicesResponse>("{\"entries\": 100000000000, \"dataSize\": 100000000000, \"fileSize\": 100000000000}"); });
```

Reply:
Perfect, did not know about it. I'll change that.